pkg/install/install.go

```diff
@@ -132,7 +132,7 @@ func DeploymentIsReady(factory client.DynamicFactory, namespace string) (bool, e
 	// declare this variable out of scope so we can return it
 	var isReady bool
 	var readyObservations int32
-	err = wait.PollImmediate(time.Second, time.Minute, func() (bool, error) {
+	err = wait.PollImmediate(time.Second, 3*time.Minute, func() (bool, error) {
```

Comment:
Was this increase because the 1 minute wait wasn't long enough to get everything installed? I think this should be called out as a separate line in the release notes.

Reply:
Remember when we had that Zoom session in December and we thought the timeout problem was happening because of Docker? And the timeout would happen on your laptop but not your server? I believe this timeout value was the problem.

Increasing the time doesn't mean it will necessarily take longer. The polling happens every second (the first argument), up to the timeout time (second argument). However, if the desired conditions aren't met by 1 minute, it will wait up to 3 minutes.

I'll go ahead and add the timeout change to the release notes.